pkgs/applications/misc/postrunner/Gemfile

```diff
@@ -0,0 +1,2 @@
+source 'https://rubygems.org'
+gem 'postrunner'
```

Comment:
Can we pin the version here?

Reply:
I pinned the version there. But I think this violates an assumption of the bundler-update-script (see pkgs/development/ruby-modules/bundler-update-script/default.nix).
But  bundler-update-script is not documented and I am still a novice NixOS user.
I would appreciate if @nicknovitski could answer this question.
I would suggest to add
in https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/ruby.section.md#packaging-executables-that-require-wrapping 
a few lines to mention the purpose of bundler-update-script and when it is called. Probably somewhere via another script at a given point for each NixOS release.

Reply:
I wrote it to follow the convention that I saw in other packages and the manual (including in that linked section), where no version specification is added to the Gemfile, since committing the lockfile pins the version anyway.